### DISCUSSION TITLE
Creating integration for `igdb.com`

### DISCUSSION THREAD

**Discussion by takov751** [inventree/InvenTree#6990] [Ideas]
I know this will be a niche idea, however i heard from retro game collectors that it would be very useful to have an inventory system. And integrating igdb.com could be almost an instant solution for these peeps. I don't have experience with creating any plugin, however when I have time I might give it a crack, unless someone finds my idea useful and would start working on it. 

I would like ask your feedback or recommendation for any other database that would be useful for retro game collectors.

**Comment by matmair** [comment]
Could certainly be interesting; I do not think any of the core devs have this use case but plugins are perfect for such scenarios
